tests/test_classification_router.py

```
@@ -0,0 +1,37 @@
+import sys
+sys.path.insert(0, 'backend')
+
+from unittest.mock import patch, MagicMock, AsyncMock
+from app.classification.classification_router import ClassificationRouter
+
+@pytest.mark.anyio
+async def test_classification_fallback_on_error():
+    """Test that classification fallback returns needs_devrel: False on error"""
+    
+    # Mock ClassificationRouter's should_process_message to force exception path
+    with patch('app.classification.classification_router.ClassificationRouter.should_process_message') as mock_method:
```

Ruff (0.15.0)

[error] Local variable `mock_method` is assigned to but never used

Remove assignment to unused variable `mock_method`

(F841)